README.md

```diff
@@ -50,7 +50,7 @@ A good way to install the `rmf` set of packages in one go is to install the one
 ```bash
 sudo apt install ros-humble-rmf-demos-gz-classic
 ```
-
+**Note:** RMF Demos cannot be installed on Humble distro because of an underlying issue with the release of a bad version of fastapi in jammy. You can install the package from source. [Read more.](https://github.com/open-rmf/rmf_demos/issues/166)
```

Comment:
I think this note is important enough to move it to the top of the RMF Demos section.
I would also remove the `sudo apt install ros-humble-rmf-demos-gz-classic` command reference and all the references to the rmf_demos_* packages and would only keep the link to RMF Demos repo for instructions.

Reply:
I agree, moving this note to the top would be more helpful.